spec-main/chromium-spec.ts

```diff
@@ -267,6 +268,38 @@ describe('command line switches', () => {
     it('should not set an invalid locale', (done) => testLocale('asdfkl', currentLocale, done));
   });
 
+  describe('--remote-debugging-pipe switch', () => {
+    it('should expose CDP via pipe', async () => {
+      const electronPath = process.execPath;
+      const appProcess = ChildProcess.spawn(electronPath, ['--remote-debugging-pipe'], {
```

Comment:
What cleans this app? It looks like this spawned application will be left open and running even after the test ends.  Should probably use an `after` or `afterEach` to ensure that the child process is killed

Reply:
Aha! I thought there was some global harness somewhere that was doing that, but I realize that i should have appProcess.kill()-d.

Reply:
(not sure how you roll and whether it should be try/finally)

Reply:
Normally we scope `appProcess` outside the test and then use an `afterEach` block to do

```js
afterEach(() => {
  if (appProcess && !appProcess.killed) appProcess.kill()
});
```